scripts/pso-collect-logs.sh

```diff
@@ -5,19 +5,15 @@ if [[ $# != 0 ]]; then
 fi
 
 KUBECTL=kubectl
+
+if [ -n "$(which oc)" ]; then
```

Comment:
I thought we didn't want to do this as a client may have both `oc` and `kubectl` if they need to access both vanilla k8s and OpenShift clusters...
I wanted to do this at first but was convinced otherwise - that is why we have the question...

Reply:
so I tested it, and looks like oc is a superset of kubectl, which works for vanilla k8s cluster too.

Reply:
Yes, it is, so if you are happy then I certainly am.
Out of interest do you collect the oc scc information in the script - probably should do as this can cause an issue if the privileged SCC is not configured correctly in OpenShift

Reply:
Sounds good, i can add it too. if it's k8s cluster we will collect something like this in the log, which does not hurt. 

root@docker-desktop:/go/src/pso.purestorage.com/k8s# oc get scc
error: the server doesn't have a resource type "scc"
root@docker-desktop:/go/src/pso.purestorage.com/k8s# oc describe scc
error: the server doesn't have a resource type "scc"

Reply:
@sdodsley added in a new commit